src/components/organisms/DMListItem/DMListItem.tsx

```diff
@@ -0,0 +1,197 @@
+    animation: '$loading 1.8s infinite ease-in-out',
+  },
+  skeletonIcon: {
+    width: '36px',
+    height: '36px',
+    borderRadius: '18px',
+  },
+  skeletonName: {
+    margin: '5px',
+    width: '60%',
+    height: '20px',
+  },
+  skeletonStatus: {
+    margin: '5px',
+    width: '35%',
+    height: '15px',
+  },
+  skeletonTime: {
+    margin: '5px',
+    width: '40%',
+    height: '20px',
+  },
+  skeletonTypo: {
+    margin: '5px',
+    width: '100%',
+    height: '40px',
+  },
+  skeletonButton: {
+    margin: '0.25em',
+    padding: '5px 15px',
+    borderRadius: '4px',
+    width: '50px',
+    height: '26px',
+  },
+});
+
+export const DMListItemSkeleton = () => {
+  const classes = useSkeletonStyles();
+  return (
+    <ListItem>
+      <Grid className={`${classes.root}`} item container justifyContent="space-around" alignItems="center">
+        <Grid item container justifyContent="center" alignItems="center" xs={1}>
+          <div className={`${classes.skeletonIcon} ${classes.skeleton}`}> </div>
+        </Grid>
+        <Grid item container justifyContent="center" alignItems="center" xs={2} direction="column">
+          <div className={`${classes.skeletonName} ${classes.skeleton}`}> </div>
+          <div className={`${classes.skeletonStatus} ${classes.skeleton}`}> </div>
+        </Grid>
+        <Grid item container justifyContent="center" alignItems="center" xs={1}>
+          <div className={`${classes.skeletonTime} ${classes.skeleton}`}> </div>
+        </Grid>
+        <Grid item container justifyContent="center" alignItems="center" xs={4}>
+          <div className={`${classes.skeletonTypo} ${classes.skeleton}`}> </div>
+        </Grid>
+        <Grid item container justifyContent="flex-end" alignItems="center" xs={1}>
+          <div className={`${classes.skeletonButton} ${classes.skeleton}`}> </div>
+        </Grid>
+      </Grid>
+    </ListItem>
+  );
+};
+
+// FIXME: Dialog, setOpen 구현하기
+type DMListItemProps = {
+  info: MessageType,
+  // eslint-disable-next-line no-unused-vars
+  setOpen: (value: boolean) => void,
+  // eslint-disable-next-line no-unused-vars
+  setDialog: (value: DialogProps) => void,
+};
+
+const makeDateString = (date: Date) => {
+  const now = new Date();
+  const today = `${now.getFullYear()}.${now.getMonth() + 1}.${now.getDate()}`;
+  const messageDate = `${date.getFullYear()}.${date.getMonth() + 1}.${date.getDate()}`;
+  if (today === messageDate) return `${date.getHours()}:${date.getMinutes()}`;
+  return `${messageDate} ${date.getHours()}:${date.getMinutes()}`;
+};
+
+const handleClickToProfile = () => {
+  // FIXME: 해당 유저의 프로필로
+};
+
+const handleClickToDM = () => {
+  // FIXME: 해당 유저의 DM chat 입장
+};
+
+const DMListItem = ({
+  // eslint-disable-next-line no-unused-vars
+  info, setOpen, setDialog,
+}: DMListItemProps) => {
+  const { user, content, createdAt } = info;
+  const { status } = user;
+  const dateStr = makeDateString(createdAt);
+  const classes = useStyles({ status });
```

Comment:
```suggestion
  const dateStr = makeDateString(createdAt);
  const classes = useStyles({ status: user.status });
```
> 또한 status를 useStyles에 넣기 위해 status만 따로 구조분해할당해서 사용했고 나머지는 user.avatar처럼 객체 프로퍼티 접근했습니다. useStyles(user.status)는 적용이 안됐고 왜 안되는 지 아직까지 의문입니다.

[이 라인](https://github.com/404-DriverNotFound/frontend-b/pull/91/files#diff-32cdbba47ef831f15b435a1c256280962c6d21efa0b89fbb9b91e7392229287eR12)에서 StyleProps를 `status`를 프로퍼티로 가지는 객체 type으로 설정했기 때문에 당연히 `user.status`만 인자로 넘겨줄 수는 없습니다. 구조분해할당을 하지 않아도 위의 suggestion처럼 작성하시면 같은 방식으로 사용하실 수 있습니다. 작성하신 type과 JS에서의 객체 할당에 대한 종합적인 이해가 필요한 부분인데, 이해가 어려우시면 [이 페이지](https://ko.javascript.info/object)의 [이 항목](https://ko.javascript.info/object#ref-311)을 보고 리마인드하시면 좋을 것 같습니다.

Reply:
오... 역시 해결방법이 있었군요. 제가 이것에 대한 개념이 없었습니다. 친절하게 코드와 링크로 남겨주셔서 감사합니다.